crates/ruff/src/rules/pylint/rules/repeated_isinstance_calls.rs

```diff
@@ -74,6 +79,9 @@ pub(crate) fn repeated_isinstance_calls(
         let [obj, types] = &args[..] else {
             continue;
         };
+        if !checker.semantic_model().is_builtin("isinstance") {
+            return;
+        }
```

Comment:
@charliermarsh I'm unable to understand why was this moved here 😅

Instead of checking once, this will check for each iteration but I don't think the semantic model changes. Am I missing something here?

Reply:
It effectively delays the check until we have at least one `isinstance` call. The result should be the same, since we're `return`ing, but we avoid having to do the `is_builtin` check if none of the calls are `isinstance`.

Reply:
Oh right, understood. Thanks!